### ISSUE TITLE
Lyric support

### ISSUE THREAD

**Issue by krimin-killr21** [kabouzeid/phonograph-issue-tracker#22]
Add support for embedded lyrics. As a person who has a lot of foreign music, it's very inconvenient not to be able to see lyrics in app. I would suggest allowing a double tap on the album art to bring in lyrics over the cover in an embedded scrollview so that controls could still be accessed, though the implementation is ultimately your choice.

**Comment by Oronhyatekha** [comment]
Great job. This player is material design in perfection. But I would like to see a feature to use it on a daily basis. I sing to my songs and all my lyrics are saved within ID3tag of the mp3. Is there a chance to make a feature like to double tap on the artwork in the "now playing" screen to see the lyrics and scroll manually through it. That would be great.﻿

**Comment by kabouzeid** [comment]
Yes, but it will have to wait a bit longer. I will let you know here once I started working on that :)

**Comment by Siphonay** [comment]
Every music player I tested did only read the lyrics tagged on MP3 files, bot not on FLAC or OGG files. That'd be great if you could look into it if you ever add the lyrics feature ! :)

**Comment by NXTaar** [comment]
As additional feature, it would be great to load missing lyrics from internet. Not every song has lyrics in its own IDv3 tag

**Comment by kabouzeid** [comment]
@NXTaar this wont be possible, not from my side but for copyright issues. I know thats stupid, but thats the reason why Apollo for example got kicked out of the Play Store

**Comment by tanvir-git** [comment]
Will there be an integration of the QuickLyric app api. Which is an open source app. In this way people who want the lyric support can install QuickLyric & enjoy all the goodies. For reference shuttle music player has a nice implementation of it.
